### PR TITLE
Refactor analysis ii

### PR DESCRIPTION
**Pull request by mschwoer**
refactor dim reduction analysis

### CODE REVIEW

File: alphastats/gui/pages/04_Analysis.py [hunk 1]
```diff
@@ -44,18 +48,27 @@
...
+    plotting_options = PlottingOptions.get_values()
     method = st.selectbox(
         "Analysis",
         options=["<select>"]
         + ["------- plots -------"]
-        + list(get_plotting_options(st.session_state).keys())
+        + plotting_options
+        + [
+            key
+            for key in list(get_plotting_options(st.session_state).keys())
```
Comment (boopthesnoot) [inline alphastats/gui/pages/04_Analysis.py:59]: dumb question: why do we have a separate `plotting_options` thing again? why not just have the output of `get_plotting_options`? because we anyways have both here?

File: alphastats/plots/DimensionalityReduction.py [hunk 2]
```diff
@@ -119,7 +120,7 @@ def _pca(self):
         }
 
     def _tsne(self, **kwargs):
-        tsne = sklearn.manifold.TSNE(n_components=2, verbose=1, **kwargs)
```
Comment (boopthesnoot) [inline alphastats/plots/DimensionalityReduction.py:122]: why? :)

why not go from `sklearn.decomposition.PCA` to `PCA` too in that case? (just a question, I actually like verbose init because it's instantly clear from which package the class comes)

**Review by boopthesnoot — APPROVED** [review]
LGTM

File: alphastats/gui/pages/04_Analysis.py (hunk 1 above)
Comment (boopthesnoot) [reply alphastats/gui/pages/04_Analysis.py:59]: UPD: I see it's gone in the next one

File: alphastats/plots/DimensionalityReduction.py (hunk 2 above)
Comment (mschwoer) [reply alphastats/plots/DimensionalityReduction.py:122]: agree, will revert to verbose import in this case

**Review by JuliaS92 — APPROVED** [review]
